src/effects/native/semiparametriceq4knobeffect.cpp

```diff
@@ -0,0 +1,139 @@
+#include "effects/native/semiparametriceq4knobeffect.h"
+
+namespace {
+static const int kStartupSamplerate = 44100;
+static const double kMinCorner = 13; // Hz
+static const double kMaxCorner = 22050; // Hz
+static const double kLpfHpfQ = 0.707106781;
+static const double kSemiparametricQ = 0.4;
+static const double kSemiparametricMaxBoostDb = 8;
+static const double kSemiparametricMaxCutDb = -20;
+} // anonymous namespace
+
```

Comment:
Do we have an external reference about these values? Please add it as a source code comment.